protos/sources/vega/governance.proto

```diff
@@ -9,6 +9,15 @@ import "vega/vega.proto";
 
 option go_package = "code.vegaprotocol.io/vega/protos/vega";
 
+
+// Spot product configuration
+message SpotProduct {
+  // Base asset name.
+  string base_asset = 1;
```

Comment:
that must be a mistake? those should be the equivlent of the settlement_asset field in the FutureProduct, and so be actual IDs.

The FutureProduct have a quote_name and settlement asset because the settlement asset can / is different to the quote_name (e.g: USDT = settlement asset, quote_name = USD), but here, the base and quote asset are the actual asset being traded and should I guess be IDs?

Reply:
they are ids, I stupidly copied the comments from the spec.

Reply:
We have this in a future, which I assumed these were the spot equivalent of and is just a plain name:
```
// Product quote name.
  string quote_name = 2;
```
but yea, whatever its _supposed_ to be lets make sure the field-name/docstring reflects it properly :p